pkg/controller/config/syncer/node_agent.go

```diff
@@ -92,8 +92,6 @@ func (s *nodeAgentSyncer) ensurePodSpec() corev1.PodSpec {
 	return corev1.PodSpec{
 		Containers:  s.ensureContainersSpec(),
 		Volumes:     s.ensureVolumes(),
-		HostNetwork: true,
-		DNSPolicy:   "ClusterFirstWithHostNet", // To have DNS options set along with hostNetwork, you have to specify DNS policy explicitly to 'ClusterFirstWithHostNet'
```

Comment:
Please don't touch the node_agent in this PR, it is different. We may need the hostNetwork here. Anyway, we can talk it when it is enabled.